pandas/io/formats/style.py

```diff
@@ -986,14 +998,14 @@ def highlight_null(self, null_color="red"):
     def background_gradient(
         self,
         cmap="PuBu",
-        low=0,
-        high=0,
-        axis=0,
+        low: float = 0,
+        high: float = 0,
```

Comment:
float or int?

Reply:
float

Reply:
While mypy won't complain because of type hierarchies I think better to do an explicit `Union[int, float]` for readability

Reply:
Comment is applicable in a few places